tests/link/test_jax.py

```diff
@@ -1253,3 +1254,12 @@ def test_RandomStream():
     jax_res_2 = fn()
 
     assert np.array_equal(jax_res_1, jax_res_2)
+
+
+def test_Erf():
+    # @ericmjl: I am open to a much better implementation of this test!
+    x = scalar("x")
+    out = Erf(x)
+    fg = FunctionGraph(x, out)
+
+    compare_jax_and_py(fg, x)
```

Comment:
I think the second argument should be a numeric input with which the output of python and Jax graphs are compared.

Reply:
We should probably update the docstrings for `inputs` and even the name to something like `test_inputs`